### ISSUE TITLE
Drivers only working on half of monitor after Windows Update

### ISSUE THREAD

**Issue by Brodod** [hawku/TabletDriver#1109]
I updated to the most recent Windows 10 version last night and when I opened the tablet drivers this morning it would only let me move my cursor using my tablet on half of my monitor. I know that I am not the only one having this issue as there were multiple people that posted on r/Osugame after the Windows update having the same problem too.

I read some other posts on here about a similar problem but after reading I realized that these people could still properly use their tablet on their desktop, just not in Osu!. This problem occurs on my desktop, not just the game.

## Hardware/software information
Windows Version: 10 Home 64 bit

Tablet Driver version: v0.2.3

Tablet model: XP-Pen G640

**Comment by XZiuS** [comment]
me too model wacom CTL 472

**Comment by VeakeR** [comment]
Same here, Wacom CTL 472

**Comment by P3di4s** [comment]
I have the same problem, but with this settings i can still play, dont know why, but this works for me. Display 2 is my main Display and the one I play osu! on![Tablet Driver 2](https://user-images.githubusercontent.com/85737282/121669309-7a4d3d00-caac-11eb-9d18-55215d5515d4.jpg)
![Tablet Driver](https://user-images.githubusercontent.com/85737282/121669315-7ae5d380-caac-11eb-9ba3-7ae3cb4ca161.jpg)

**Comment by XZiuS** [comment]
![image](https://user-images.githubusercontent.com/85735479/121685765-8b1ca380-caea-11eb-90fc-f50f6b1863ca.png)
i tried this, and it works for me oon my monitor 1

**Comment by XZiuS** [comment]
![image](https://user-images.githubusercontent.com/85735479/121686036-e058b500-caea-11eb-8cc7-5e796d9a6fbc.png)
i tried this and it works for my 1st monitor
the sensitivity is weird i still cant play

**Comment by BadatHoppin** [comment]
Im currently having the same problem, Huion H430P

**Comment by Troved** [comment]
this also happened to me after the windows update and my tablet is the CTL-472. I still have no idea how to fix it hopefully its's fixed soon.

**Comment by UltayJr** [comment]
Exact same, for 2 days now.
Windows version: 10 pro, 2004 version and 19041.1052 build
Tablet driver version: 0.2.3
Tablet model: CTL-480
The latest windows update is the problem, on a 3-monitor setup, the same problem, my area is a square at the middle of my main screen and can't be modified
![image](https://user-images.githubusercontent.com/54221199/121708771-2c4e2e80-cad8-11eb-86b9-0c52e6e1cb6d.png)

**Comment by sambouiss** [comment]
seeting 

> I have the same problem, but with this settings i can still play, dont know why, but this works for me. Display 2 is my main Display and the one I play osu! on![Tablet Driver 2](https://user-images.githubusercontent.com/85737282/121669309-7a4d3d00-caac-11eb-9d18-55215d5515d4.jpg)
> ![Tablet Driver](https://user-images.githubusercontent.com/85737282/121669315-7ae5d380-caac-11eb-9ba3-7ae3cb4ca161.jpg)

This worked for me god bless

**Comment by neckztik** [comment]
There is a fix for this, all you do is you need to keep your area the same and make the screen map to both your monitors.
![image](https://user-images.githubusercontent.com/48533906/121724581-ecb62100-ca9c-11eb-8abd-ed48bc56bda9.png)
Hope this gets fixed, but it honestly probably won't.

**Comment by TheMinecar** [comment]
Same thing is happening to me except I'm on a VEIKK S640.

**Comment by Instellate** [comment]
> 
> 
> I updated to the most recent Windows 10 version last night and when I opened the tablet drivers this morning it would only let me move my cursor using my tablet on half of my monitor. I know that I am not the only one having this issue as there were multiple people that posted on r/Osugame after the Windows update having the same problem too.
> 
> I read some other posts on here about a similar problem but after reading I realized that these people could still properly use their tablet on their desktop, just not in Osu!. This problem occurs on my desktop, not just the game.
> ## Hardware/software information
> 
> Windows Version: 10 Home 64 bit
> 
> Tablet Driver version: v0.2.3
> 
> Tablet model: XP-Pen G640

Hey everyone that reads this. I will just tell this. The issue is made by the new windows security update. I uninstalled the update and the drivers worked fine. I do not recommend to uninstall it if you dont know what you are doing. Until that, just wait until its fixed or uninstall the security update if you know what you are doing!

**Comment by SnowLire** [comment]
#1087

**Comment by SnowLire** [comment]
> Hey everyone that reads this. I will just tell this. The issue is made by the new windows security update. I uninstalled the update and the drivers worked fine. I do not recommend to uninstall it if you dont know what you are doing. Until that, just wait until its fixed or uninstall the security update if you know what you are doing!

this will never get fixed btw. hawku has abandoned this project years ago

**Comment by AyanoNK** [comment]
> t y

I will try to peek at the files and see if I can fix it. I'll make a branch if I can fix it.

**Comment by SnowLire** [comment]
> I will try to peek at the files and see if I can fix it. I'll make a branch if I can fix it.

why bother, just use OpenTabletDriver

**Comment by AyanoNK** [comment]
> why bother, just use OpenTabletDriver
:(

**Comment by Troved** [comment]
how do you fix this please I want to play

**Comment by SnowLire** [comment]
> how do you fix this please I want to play 

you don't. use OpenTabletDriver

**Comment by danielkemp017** [comment]
Quick Tip guys a current workaround i found was to set ur area as per normal but set the 'set area' dropdown to full desktop 
![image](https://user-images.githubusercontent.com/50493375/121785625-b1a61180-cbb2-11eb-8e8f-fc74e790bb8a.png)
after that go into the Misc tab then set the desktop to whatever size is ur native in my case 1920*1080 
![image](https://user-images.githubusercontent.com/50493375/121785640-d39f9400-cbb2-11eb-985a-976202880d3b.png)
Hope this helps until a more permanent fix is done

**Comment by lukemagdalin** [comment]
> Quick Tip guys a current workaround i found was to set ur area as per normal but set the 'set area' dropdown to full desktop
> ![image](https://user-images.githubusercontent.com/50493375/121785625-b1a61180-cbb2-11eb-8e8f-fc74e790bb8a.png)
> after that go into the Misc tab then set the desktop to whatever size is ur native in my case 1920*1080
> ![image](https://user-images.githubusercontent.com/50493375/121785640-d39f9400-cbb2-11eb-985a-976202880d3b.png)
> Hope this helps until a more permanent fix is done

Thanks, This works for the time being.

**Comment by Troved** [comment]
> > how do you fix this please I want to play
> 
> you don't. use OpenTabletDriver

I tried using it but it keeps telling me to download .NET and I already did but it still tells me to download it. 
![image](https://user-images.githubusercontent.com/77676082/121795240-1c9a2b80-cc39-11eb-88ab-8da7d3b094c7.png)

**Comment by SnowLire** [comment]
> I tried using it but it keeps telling me to download .NET and I already did but it still tells me to download it
> ![image](https://user-images.githubusercontent.com/77676082/121795240-1c9a2b80-cc39-11eb-88ab-8da7d3b094c7.png)
> 
> I already pressed yes and downloaded it
> and nothing happens.

was the file named windowsdesktop-5.0.7-win-x64? if not then you probably downloaded the wrong one. You have to download .NET 5 Desktop Runtime x64

**Comment by Instellate** [comment]
> > Hey everyone that reads this. I will just tell this. The issue is made by the new windows security update. I uninstalled the update and the drivers worked fine. I do not recommend to uninstall it if you dont know what you are doing. Until that, just wait until its fixed or uninstall the security update if you know what you are doing!
> 
> this will never get fixed btw. hawku has abandoned this project years ago

Nah. Microsoft will fix it cause its a problem with multiple drivers. So we just need to wait for them to fix it next week

**Comment by XZiuS** [comment]
> > > how do you fix this please I want to play
> > 
> > 
> > you don't. use OpenTabletDriver
> 
> I tried using it but it keeps telling me to download .NET and I already did but it still tells me to download it
> ![image](https://user-images.githubusercontent.com/77676082/121795240-1c9a2b80-cc39-11eb-88ab-8da7d3b094c7.png)
> 
> I already pressed yes and downloaded it
> and nothing happens.
 same here

**Comment by XZiuS** [comment]
> > > how do you fix this please I want to play
> > 
> > 
> > you don't. use OpenTabletDriver
> 
> I tried using it but it keeps telling me to download .NET and I already did but it still tells me to download it
> ![image](https://user-images.githubusercontent.com/77676082/121795240-1c9a2b80-cc39-11eb-88ab-8da7d3b094c7.png)
> 
same here

**Comment by SnowLire** [comment]
[.NET 5 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-5.0.7-windows-x64-installer)

**Comment by Troved** [comment]
> [.NET 5 Desktop Runtime](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-5.0.7-windows-x64-installer)

ty but after this how do u fit it in one monitor only? i have two of em and it only works on my second one rn
![image](https://user-images.githubusercontent.com/77676082/121826249-f4fe9e00-cce0-11eb-8e88-4b4d55392986.png)

In my old devocub one it has an option
![image](https://user-images.githubusercontent.com/77676082/121826266-13fd3000-cce1-11eb-86e0-3147180d3f9f.png)
